addons/mail/models/mail_thread.py

```diff
@@ -2135,6 +2135,41 @@ def _message_create(self, values_list):
     # NOTIFICATION API
     # ------------------------------------------------------
 
+    @api.model
+    def discard_notifications(self, notification_type):
```

Comment:
Would name it ``notify_cancel_by_type`` . We have to keep it public (aka not starting with "_" ) as it is called by JS.